doc/fortunes.tips

```diff
@@ -79,7 +79,7 @@ This code was intentionally left blank, try 'e asm.arch = ws'
 Thanks for using radare2!
 give | and > a try piping and redirection
 Run .dmm* to load the flags of the symbols of all modules loaded in the debugger
-Use V! to enter into the visual panels mode (dwm style)
+Use V! to enter visual panels mode
```

Comment:
this has "changed" to something simpler, which i think is good to document:

* v (lowercase) starts panels
* V (uppercase) start visual mode

The V! is still a thing, but imho its easier to just tell people to use v or V and ! to toggle between panels/visual

Reply:
Agreed.